### PR TITLE
Feature/notify 12 log dating

### PR DESCRIPTION
**Pull request by mhaswell-bcgov**

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
...
+## Deploying to local Kubernetes cluster (Docker Desktop)
+1. Ensure Docker Desktop has Kubernetes enabled (Settings > Kubernetes > Enable Kubernetes).
+2. Ensure you are on the `docker-desktop` context (should be the current context when running `kubectl config get-contexts`).
+3. From project root, deploy to the Kubernetes cluster using the Helm command `helm install local -f ./helm/values.local.yaml ./helm/`.
+   - `local` in the example is the release name and can be changed to any name you want.
+   - `values.local.yaml` is a values file that changes configurations so the deployment will work locally.
+   - To add any other custom values (to change the image version, for example), add another `-f` flag pointing to your values yaml file.
+4. To undo a deployment, run the command `helm delete local`.
+   - If you used a different release name in the previous step, replace `local` with that release name.
+
```
Comment (robrien-bcgov) [inline helm/README.md:16]: this seems easy to follow. I like numbered lists with actual commands in them

File: helm/values.yaml [hunk 2]
```diff
@@ -202,14 +202,25 @@ fluentbit:
     service: ""
     inputs: |
       [INPUT]
-          Name tail
-          Path /tmp/logs/*.log
-          Tag app.access.log
-          Mem_Buf_Limit 5MB
+          Name            tail
+          Path            /tmp/logs/*.log
```
Comment (robrien-bcgov) [inline helm/values.yaml:206]: why all the whitespace changes?

File: helm/values.yaml [hunk 3]
```diff
@@ -202,14 +202,25 @@ fluentbit:
...
+          Tag             app.access.log
+          Mem_Buf_Limit   5MB
           Skip_Long_Lines Off
-    filters: ""
+    filters: |
+      [FILTER]
+          Name  Lua
+          Match *
+          call  append_tag
+          code  function append_tag(tag, timestamp, record) new_record = record new_record["datestamp"] = os.date("%Y%m%d", timestamp) return 1, timestamp, new_record end
```
Comment (robrien-bcgov) [inline helm/values.yaml:215]: cool - Lua code!

seems to be appending a tag to each record containing a datestamp.

**Review by robrien-bcgov — APPROVED** [review]
seems reasonable, though I lack context on this.

File: helm/values.yaml (hunk 2 above)
Comment (mhaswell-bcgov) [reply helm/values.yaml:206]: I thought it would be easier to read if the values were aligned with each other.

File: helm/values.yaml (hunk 3 above)
Comment (mhaswell-bcgov) [reply helm/values.yaml:215]: That's right. It's pretty hacky but I'm just wanting to finish this thing up even if it isn't done in the best way.

**Pull request merged**